docs/current/sdk/elixir/043817-install.md

```diff
@@ -0,0 +1,19 @@
+---
+slug: /sdk/elixir/043817/install
+---
+
+# Installation
+
+:::note
+This Dagger Elixir SDK requires Elixir 1.14 or later with Erlang OTP 23 or later. Using Erlang OTP version 25 is recommended.
+:::
+
+Install the Dagger Elixir SDK in your project by adding this package into `mix.exs` in `deps` function:
+
+```elixir
+def deps do
+  [
+    {:dagger, "~> 0.8"}
```

Comment:
I think `0.8` should replace by something and tells in this document that they should see release page for the correct version.

Reply:
The simplest thing might be to add a link to https://hex.pm/packages/dagger and ask users to find the latest version there. This also looks like a good option:

```console
mix hex.info dagger
Dagger SDK for Elixir

Config: {:dagger, "~> 0.8.4"}
Releases: 0.8.4, 0.8.2

Licenses: Apache-2.0
Links:
  Changelog: https://github.com/dagger/dagger/releases/tag/sdk%2Felixir%2Fv0.8.4
  GitHub: https://github.com/dagger/dagger
```

Having said that, the problem is that we don't know in what way the next version will change. We know that these docs work as expected with `~> 0.8`, so perhaps we should bump this version whenever there is a new minor, and we have reviewed that everything works as expected.

We could go further and add automation which checks that the examples work as expected with the local `dev` version, before it gets published. We then could bump this part of the `sdk:elixir:bump`. For now, this sounds overkill & maybe leaving it as is would be best.

Reply:
modifying it in the `sdk:elixir:bump` is a great idea! I'll try after this PR is merged. :)